draft-ietf-aipref-vocab.md

```diff
@@ -203,11 +203,144 @@ If arrangements, such as contracts exist that explicitly permit the use of that
 
 The vocabulary does not preclude the use of other specific categories. Any statement of preference based on this vocabulary shall not be interpreted as restricting the use of the work(s) strictly for the purpose of search and discovery as long as no restriction is declared through search-specific means such as {{!RFC9309}}.
 
-## Vocabulary Extensions
+## Vocabulary Extensions {#vocab-extension}
 
 Systems referencing the vocabulary must not introduce additional categories that include existing categories defined in the vocabulary or otherwise include additional hierarchical relationships.
 
 
+# Serialization Format {#format}
+
+This section defines an exemplary serialization format for preferences.
+The format describes how the abstract model could be turned into a Unicode string or sequence of bytes.
+
+The format relies on the Dictionary type defined in {{Section 3.2 of !FIELDS=RFC9651}}.
+The dictionary keys correspond to usage categories
+and the dictionary values correspond to explicit preferences.
+
```

Comment:
Add something here about the values, `y` and `n`, so that it is clear what they represent.  Or just refer to the later section